web_app_drawer/views/web.xml

```diff
@@ -0,0 +1,252 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+    Copyright 2016 LasLabs Inc.
+    License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+-->
+
+<odoo>
+
+    <template id="webclient_bootstrap"
+              inherit_id="web.webclient_bootstrap"
+              name="App Drawer - Web Client"
+              >
+        
+        <xpath expr="//div[@class='oe_leftbar']" position="attributes">
+            <attribute name="class">hidden</attribute>
+        </xpath>
+        
+        <xpath expr="//t[@t-set='head']" position="inside">
+            <meta charset="utf-8" />
+            <meta http-equiv="cleartype" content="on" />
+            <meta name="MobileOptimized" content="320" />
+            <meta name="HandheldFriendly" content="True" />
+            <meta name="apple-mobile-web-app-capable" content="yes" />
+            <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no" />
+        </xpath>
+        
+        <xpath expr="//nav[@id='oe_main_menu_navbar']" position="replace">
+            
+            <t t-set="body_classname" t-value="'drawer drawer--left'" />
+                
+            <header role="banner">
+                <nav id="odooAppDrawer" class="app-drawer-nav drawer-nav" role="navigation">
+                    <t t-call="web.menu" />
+                </nav>
+                
+                <nav class="navbar navbar-default main-nav"
+                     role="navigation"
+                     groups="base.group_user,base.group_portal"
+                     >
+                    <div class="container-fluid">
+                        
+                        <div class="navbar-header">
+                            
+                            <a href="#"
+                               class="drawer-toggle pull-left navbar-collapse collapse btn btn-default app-drawer-toggle"
+                               accesskey="A"
+                               >
+                                <span class="sr-only">Toggle Navigation</span>
+                                <i class="fa fa-th fa-2x app-drawer-icon-open" />
+                            </a>
+                            
+                            <button type="button"
+                                    class="app-drawer-toggle drawer-toggle pull-left navbar-toggle collapsed"
+                                    >
+                                <span class="sr-only">Toggle Navigation</span>
+                                <i class="fa fa-th fa-2x app-drawer-icon-open" />
+                            </button>
+                            
+                            <a href="#" class="app-drawer-icon-close btn btn-danger">
+                                <i class="fa fa-times fa-2x drawer-toggle" />
```

Comment:
Well... actually I think it would be much better to just put this icon as a [chevron-left](http://fontawesome.io/icon/chevron-left/) just before `#appDrawerAppPanelHead>h3`